src/map_io/map_players_view_packet.cc

```diff
@@ -161,10 +161,10 @@ namespace {
 
 #define CHECK_TRAILING_BYTES(file, filename)                                                       \
 	if (!(file).end_of_file())                                                                      \
-		throw GameDataError(                                                                         \
-		   "MapPlayersViewPacket::read: player %u:"                                                  \
-		   "Found %" PRIuS " trailing bytes in \"%s\"",                                              \
-		   plnum, static_cast<long unsigned int>((file).get_size() - (file).get_pos()), filename);
+		throw GameDataError("MapPlayersViewPacket::read: player %u:"                                 \
+		                    "Found %" PRIuS " trailing bytes in \"%s\"",                             \
+		                    plnum, static_cast<size_t>((file).get_size() - (file).get_pos()),        \
```

Comment:
Can you fix all the plnum and not just the one? Search the file for `MapPlayersViewPacket::read: player`. Thanks :)

Reply:
I have done so.

Reply:
Excellent, thanks!